cfgov/legacy/templates/front/base_update.html

```diff
@@ -86,36 +109,43 @@
     {% endblock %}
 
     <!--[if lt IE 9]>
-    <link rel="stylesheet" href="{% static 'css/header.nonresponsive.css' %}"/>
-    <link rel="stylesheet" href="{% static 'css/footer.nonresponsive.css' %}"/>
+    <link rel="stylesheet" href="{{ STATIC_PREFIX }}css/header.nonresponsive.css"/>
+    <link rel="stylesheet" href="{{ STATIC_PREFIX }}css/footer.nonresponsive.css"/>
     <![endif]-->
 
     <!--[if gt IE 8]><!-->
-    <link rel="stylesheet" href="{% static 'css/header.css' %}"/>
```

Comment:
Okay, let me know how it looks on build. I just updated it to this because I saw other scripts being called this way in the file.